lib/bbbevents.rb

```diff
@@ -1 +1,6 @@
 require "bbbevents/base"
+require 'bbbevents/version'
```

Comment:
No reason to make this change, just leave the current structure with includes from `base.rb` as-is

Reply:
moved to `base.rb`